README.md

```diff
@@ -12,8 +12,9 @@ Configure it as shown:
 ![Proxy config](https://i.imgur.com/TaEWjkB.png)
 
 Run: `git clone https://github.com/AnanthVivekanand/spotify-adblock-macos.git && cd spotify-adblock-macos`  
-Run: `npm i http-proxy micromatch`  
-Run: `node index.js`  
+Run: `npm i`
+Run: `export PORT=8080`  
```

Comment:
This isn't formatted correctly. If you could move the `export PORT=8080` onto a new line, that would be great.
![image](https://user-images.githubusercontent.com/40674932/77809695-d28e4f80-705e-11ea-806c-44c21d1a4956.png)